docs/en/sql-reference/sql-functions/Window_function.md

```diff
@@ -134,304 +165,256 @@ select stock_symbol, closing_date, closing_price,
-| 5  | odd      | 5              |
-| 7  | odd      | 7              |
-| 9  | odd      | 8              |
-+----+----------+----------------+
-```
+For example, `12.87500000` in the first row is the average value of closing prices on "2014-10-02" (`12.86`), its previous day "2014-10-01" (null), and its following day "2014-10-03" (`12.89`).
 
 ### COUNT()
 
-Syntax:
+Calculates the total number of rows that meet the specified conditions in a give window.
+
+**Syntax:**
 
 ```SQL
 COUNT(expr) [OVER (analytic_clause)]
 ```
 
-Example:
+**Examples:**
 
-Count the occurrence of x from the current row to the first row.
+Count the occurrence of math scores that are greater than 90 from the current row to the first row in the math partition. This example uses the data in the [Sample table](#window-function-sample-table) `scores`.
 
 ```SQL
-select x, property,
-    count(x)
+select *,
+    count(score)
         over (
-            partition by property
-            order by x
+            partition by subject
+            order by score
             rows between unbounded preceding and current row
-        ) as 'cumulative total'
-from int_t where property in ('odd','even');
+        ) as 'score_count'
+from scores where subject in ('math') and score > 90;
 ```
 
 ```plaintext
-+----+----------+------------------+
-| x  | property | cumulative count |
-+----+----------+------------------+
-| 2  | even     | 1                |
-| 4  | even     | 2                |
-| 6  | even     | 3                |
-| 8  | even     | 4                |
-| 10 | even     | 5                |
-| 1  | odd      | 1                |
-| 3  | odd      | 2                |
-| 5  | odd      | 3                |
-| 7  | odd      | 4                |
-| 9  | odd      | 5                |
-+----+----------+------------------+
++------+-------+---------+-------+-------------+
+| id   | name  | subject | score | score_count |
++------+-------+---------+-------+-------------+
+|    6 | amber | math    |    92 |           1 |
+|    3 | jack  | math    |    95 |           2 |
++------+-------+---------+-------+-------------+
 ```
 
 ### CUME_DIST()
 
 The CUME_DIST() function calculates the cumulative distribution of a value within a partition, indicating its relative position as a percentage of values less than or equal to the value in the current row. With a range of 0 to 1, it's useful for percentile calculations and data distribution analysis.
 
-Syntax:
+This function is supported from v3.2.
+
+**Syntax:**
 
 ```SQL
 CUME_DIST() OVER (partition_by_clause order_by_clause)
 ```
 
-**This function should be used with ORDER BY to sort partition rows into the desired order. Without ORDER BY, all rows are peers and have value N/N = 1, where N is the partition size.**
+**This function must be used with ORDER BY to sort partition rows into the desired order. Without ORDER BY, all rows are peers and have value `N/N = 1`, where `N` is the partition size.**
```

Comment:
Without ORDER BY, all rows are peers and have value `N/N = 1`, where `N` is the partition size.

Have the value 

Also, I am not sure what does this is supposed to mean. I didn't find the similar description in CN doc